src/runtime_src/core/common/message.cpp

```diff
@@ -277,5 +278,21 @@ send(severity_level l, const char* tag, const char* msg)
     dispatcher->send(l, tag, msg);
   }
 }
+
+void
+send(severity_level l, const char* tag, const char* format, va_list args)
+{
+  static auto verbosity = xrt_core::config::get_verbosity();
+  if (l > (xrt_core::message::severity_level)verbosity) {
+    return;
+  }
```

Comment:
Remove the braces, leave blank line after return.

Reply:
done